packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -27,6 +28,8 @@ if (!disabledInstrumentations.has('fetch')) {
 const HOOK_SYMBOL = Symbol('hookExportsMap')
 // TODO: make this more efficient
 
+checkRequireCache()
```

Comment:
TODO: Need to wrap this in a debug check. I don't think config is ready at this point?

Reply:
Oh, duh, of course, config can't possibly be ready before init, as init is used to configure the tracer.